modules/imgcodecs/include/opencv2/imgcodecs.hpp

```diff
@@ -110,9 +110,11 @@ returns an empty matrix ( Mat::data==NULL ). Currently, the following file forma
 -   JPEG 2000 files - \*.jp2 (see the *Notes* section)
 -   Portable Network Graphics - \*.png (see the *Notes* section)
 -   WebP - \*.webp (see the *Notes* section)
--   Portable image format - \*.pbm, \*.pgm, \*.ppm (always supported)
+-   Portable image format - \*.pbm, \*.pgm, \*.ppm \*.pxm, \*.pnm (always supported)
```

Comment:
do we really support pxm and pnm? I thought, pxm is a common term for pbm, pgm and ppm. And I don't know what pnm is